openedx/features/caliper_tracking/transformers/bookmark_transformers.py

```diff
@@ -13,7 +14,34 @@ def edx_bookmark_added(current_event, caliper_event):
 
 
 def edx_bookmark_accessed(current_event, caliper_event):
-    print('edx_bookmark_accessed')
+    """
```

Comment:
The unit tests are failing on this. Please review it and modify accordingly.

Reply:
Tests passing.